using_images/s2i_images/ruby.adoc

```diff
@@ -96,6 +96,13 @@ and ruby gem installation. The application assets are only compiled if
 Asset compilation only happens when the application runs in a production
 environment. Therefore, you can use this variable when assets have already been
 compiled.
+
+|`*PUMA_MIN_THREADS*`, `*PUMA_MAX_THREADS*`
+|These variables indicate the minimum and maximum threads that will be available
+|These variables indicate the minimum and maximum threads that will be available in link:https://github.com/puma/puma[Puma]'s thread pool.
+
+|`*PUMA_WORKERS*`
+|This variable indicate the number of worker processes that will be launched. See documentation on Puma's link:https://github.com/puma/puma#clustered-mode[clustered mode].
```

Comment:
s/indicate/indicates/

Also you can remove "This variable/These variables" from all these descriptions, so it would just read "Indicates the number of....."